poc/common.sage

```diff
@@ -8,10 +8,10 @@ import struct
 
 # If set, then test vectors will be generated. A fixed source of randomness is
 # used for `gen_rand()`.
-TEST_VECTOR = False
+TEST_VECTOR = True
 
 # Document version, reved with each draft that contains breaking changes.
-DRAFT = '03'
+DRAFT = '04'
```

Comment:
Let's revert this. Bumping the version is usually a chore saved until just before cutting the draft.